doc/visualizations.rst

```diff
@@ -34,9 +42,9 @@ vector machine:
 The returned `svc_disp` object allows us to continue using the already computed
```

Comment:
What do you think about adding the `from_predictions` example here. 

We could just say e.g., "If you already have the prediction values, you could instead use `from_predictions` to do the same thing: ...

Reply:
Done